executor/runtime/docker/docker.go

```diff
@@ -1587,6 +1579,27 @@ func (r *DockerRuntime) k8sContainerToDockerConfigs(v1Container v1.Container, ma
 	return dockerContainerConfig, dockerHostConfig, dockerNetworkConfig
 }
 
+func (r *DockerRuntime) computeDockerEntrypoint(v1Container v1.Container) []string {
+	// What docker calls "entrypoint", k8s calls "command", but in addition, we prepend tinit
```

Comment:
The comment says `tinit` but the code appears to prepend `/sbin/docker-init` (also, should `tinit` be `tini`)?